rfcs/0025-nix-core-team.md

```diff
@@ -0,0 +1,82 @@
+---
+feature: nix-core-team
+start-date: 2018-01-31
+author: Graham Christensen
+co-authors: Daniel Peebles, Eelco Dolstra, Peter Simons, Shea Levy, Vladimír Čunát
+related-issues:
+---
+
+# Summary
+[summary]: #summary
+
+Create an experimental Nix Core Team to help lead the direction of
+Nix. This RFC may not be perfect, and we don’t have good answers to
+all the possible questions, but let’s try it.
+
+# Motivation
+[motivation]: #motivation
+
+ - Improve visibility in to how the project operates
+ - Distribute the work Eelco has been doing across more people
+ - "Unstuck" pull requests which are sitting idle
+ - Provide a more diverse group of experiences when evaluating changes
+   to core Nix
+
+# Detailed design
+[design]: #detailed-design
+
+## This team will:
+
+ - Evaluate larger features being proposed to Nix
+ - Serve as a second opinion on Nix changes that Eelco doesn't
+   otherwise see the value to
```

Comment:
Is the longer term goal here to distribute consensus for vision/scope changes of NixOS to a wider group than just one person? Can we rephrase it so it isn't about the negative perception of a specific person? Perhaps: "Will approve Nix changes and steering for the long term vision"?

Reply:
Further future could be wished-for here, but I understand the intention as an incremental process: let's try this first and after some time we'll better see where to go next.